Tests/W3CTestSuite/svg/__pull_request-492-01.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Very nice :) Could you also add this to the passing tests (both the csv and the txt files, I may unify these later...)?